crates/binstalk-downloader/src/remote/resolver.rs

```diff
@@ -33,12 +33,12 @@ impl Resolve for TrustDnsResolver {
 fn new_resolver() -> Result<TokioAsyncResolver, BoxError> {
     #[cfg(unix)]
     {
-        info!("Using system DNS resolver configuration");
+        debug!("Using system DNS resolver configuration");
         Ok(TokioAsyncResolver::tokio_from_system_conf()?)
     }
     #[cfg(windows)]
     {
-        info!("Using custom DNS resolver configuration");
+        debug!("Using custom DNS resolver configuration");
```

Comment:
this one?

Reply:
Nope, I was referring to the tracing import under `cfg(windows)`.


Sorry should provide the line number, it's on line 10.